chapter_11_ansible.asciidoc

```diff
@@ -218,6 +232,9 @@ At my registrar, the control screens looked a bit like <<registrar-control-scree
 .Domain setup
 image::images/gandi_add_dns_a_record.png["Registrar control screen for adding a DNS record"]
 
+// CSANAD: due to technical reasons, I cannot check on Digital Ocean whether this
+//         screenshot needs to be update. Please, someone else have a look!
```

Comment:
*updated

Reply:
i litterally just made that screenshot last month so it's ok.